_includes/software.html

```diff
@@ -0,0 +1,21 @@
+<table class="table">
```

Comment:
This file needs to be untracked (`git rm _includes/software.html') and added to the .gitignore (Under make targets)

Reply:
This is no longer relevant, since software.html is no longer auto-generated

Reply:
Can _includes/software.html be consolidated into software.html then?

Reply:
yes